devicetypes/smartthings/zigbee-metering-plug.src/zigbee-metering-plug.groovy

```diff
@@ -175,6 +187,14 @@ private int getEnergyDiv() {
     }
 }
 
+BigDecimal calculateDelta (BigDecimal currentEnergy, Map previousMap) {
+        if (previousMap == null || previousMap ['energy'] == null) {

```

Comment:
This construction would be more concise

if (previousMap?.'energy' == null) {
                result = 0;
}

Reply:
I'll do so.